test-configs.yaml

```diff
@@ -30,6 +30,10 @@ file_systems:
     type: buildroot
     nfs: '{arch}/base/rootfs.tar.xz'
 
+  buildroot_baseline_ramdisk:
+    type: buildroot
+    ramdisk: '{arch}/baseline/rootfs.cpio.gz'
```

Comment:
Where is this ramdisk coming from?  AFAICT, we're only building the `base` variant of buildroot images.

Oh, I see in the STAGING patch how you're doing it.  So this should actually use `base` and then switch it to `baseline` in your staging patch.

Reply:
Yes, there's now also a `baseline` version being built.